library/ecp_curves.c

```diff
@@ -834,6 +870,11 @@ int mbedtls_ecp_group_load( mbedtls_ecp_group *grp, mbedtls_ecp_group_id id )
             return( LOAD_GROUP_A( brainpoolP384r1 ) );
 #endif /* MBEDTLS_ECP_DP_BP384R1_ENABLED */
 
+#if defined(MBEDTLS_ECP_DP_SM2P256V1_ENABLED)
```

Comment:
Please put this after the BP512R1 case. Having a consistent order makes it easier to maintain the code.

Reply:
OK